components/desktop/mate/mate-applets/Makefile

```diff
@@ -76,8 +76,6 @@ REQUIRED_PACKAGES += library/glib2
 REQUIRED_PACKAGES += library/libgtop
 REQUIRED_PACKAGES += library/libnotify
 REQUIRED_PACKAGES += library/libxml2
-REQUIRED_PACKAGES += library/python/pygobject-3-27
```

Comment:
Are you sure that this is not build deps?

Reply:
It actually is required, I brought it back. Thanks.